src/vtok_p11/src/backend/db/object.rs

```diff
@@ -103,6 +109,15 @@ impl Attr {
         Self::CkKeyType(src.to_le_bytes())
     }
 
+    fn from_ck_cert_type(src: pkcs11::CK_CERTIFICATE_TYPE) -> Self {
+        #[cfg(target_endian = "little")]
+        Self::CkCertType(src.to_le_bytes())
+    }
+    fn from_ck_cert_category(src: pkcs11::CK_CERTIFICATE_CATEGORY) -> Self {
+        #[cfg(target_endian = "little")]
```

Comment:
I don't think this compiles on big endian.

Reply:
True, these additions are the same as the rest of these attributes for x86_64 and aarch64 little-endian mode. We never tested big endian builds or run-time.